Installer for Windows CPU.bat

```diff
@@ -0,0 +1,8 @@
+# Install python 3.10

```

Comment:
`#` is not comment syntax for windows bat file.
Instead, use rem command
`@rem Install python 3.10`

Reply:
Fixed